geometry_msgs/Point pos
geometry_msgs/Point vel
geometry_msgs/Point accel